packages/core/core-flows/src/cart/workflows/complete-cart.ts

```diff
@@ -215,15 +215,30 @@ export const completeCartWorkflow = createWorkflow(
           .map((adjustment) => adjustment.code)
           .filter(Boolean)
 
+        const shippingAddress = cart.shipping_address
```

Comment:
We need a transformer to check if cart.something is present, otherwise at this point in the workflow composition, it will always be true.